### PR TITLE
Hotfix request manager initialization

### PR DESCRIPTION
**Pull request by Thodor12**
See #10281

### CODE REVIEW

**Review by someaddons — DISMISSED** [review]

**Comment by Thodor12** [comment]
Wait, found another bug

File: src/main/java/com/minecolonies/core/colony/Colony.java [hunk 1]
```diff
@@ -1349,6 +1348,10 @@ public ServerLevel getWorld()
     @Override
     public IRequestManager getRequestManager()
     {
+        if (requestManager == null)
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/core/colony/Colony.java:1352]: Also not sure how guaranteed this was.

Comment (Raycoms) [reply src/main/java/com/minecolonies/core/colony/Colony.java:1352]: Like if it's accessed anyway midclass directly. Probably saver with the previous setup with method to set?

Comment (someaddons) [reply src/main/java/com/minecolonies/core/colony/Colony.java:1352]: the reason why we need to delay it is that during colony loading from nbt the manager had to be set later, which was done originally through a specific set call. The usages of the getter all seem to expect nonnull though, no one checks for it so should be fine :)

Comment (Thodor12) [reply src/main/java/com/minecolonies/core/colony/Colony.java:1352]: I only wonder if we do this if we shouldn't remove all direct usages of `requestManager` as those can potentially be null right now.

**Pull request merged**